src/test/java/pageComponents/HeaderGeneralPageComponent.java

```diff
@@ -36,6 +41,15 @@ public HeaderGeneralPageComponent(WebDriver driver) {
     private WebElement signUpBtn;
 
     public HeaderGeneralDropDownMenuPageComponent userMenu()   {
+        wait = new WebDriverWait(driver, Duration.ofSeconds(15));
```

Comment:
Can you please overload this method. Also I think we don't need to use "magic variables".

Reply:
The same comment to other similar methods

Reply:
Done